integration-tests/goss/generate_goss.sh

```diff
@@ -34,22 +34,8 @@ goss a "${args[@]}" group $user foobar
 
 goss a "${args[@]}" command "echo 'hi'" foobar
 
-goss a "${args[@]}" dns --timeout 1s --server 8.8.8.8 CNAME:c.dnstest.io
```

Comment:
I think the intent of this test is to assert that goss can _generate_ these tests. Looks like the output of this test is then used as the input to a follow-on which asserts that `dns` assertion runs and passes.

Is that something that could be broken apart?

Reply:
The `goss a` command generates goss test by inspecting (in this case) the actual CNAME of c.dnstest.io. So we can't really split it in a simple way.

Reply:
Ah!

Reply:
I've created https://github.com/dnstest/dnstest.github.io/issues/1 to see whether we can get the config so we can containerise, then here it sounds like we could modify the `--server` flag to point at a running container hosting the DNS entries that we need. 

I figure that's a way to localise and decouple from an internet based remote dependency. What do you think?

In the meantime, as elsewhere, can we comment-out with a TODO?

Reply:
Yep, done.